### PR TITLE
Support path with spaces

### PR DESCRIPTION
**Pull request by MegaV0lt**
Fix for https://github.com/azlux/log2ram/issues/129#issue-707806450

### CODE REVIEW

File: log2ram [hunk 1]
```diff
@@ -36,28 +36,28 @@ syncFromDisk () {
...
-        cp -rfup $HDD_LOG/ -T $RAM_LOG/ 2>&1 | tee -a $LOG2RAM_LOG
+        cp -rfup "$HDD_LOG"/ -T "$RAM_LOG"/ 2>&1 | tee -a "$LOG2RAM_LOG"
     fi
 
 }
 
 wait_for () {
-    while ! grep -qs "$1" /proc/mounts; do
-        sleep 0.1
+    while ! mount | grep "$1" > /dev/null; do
```
Comment (zenofile) [inline log2ram:59]: Is there a specific reason not to simply use `findmnt` instead of grepping?

Comment (MegaV0lt) [reply log2ram:59]: Not realy, but is it safe to use? Is it available on all systems?

Comment (zenofile) [reply log2ram:59]: It is part of `util-linux` and `man 8 mount` has this to say:

```
   Listing the mounts
       The listing mode is maintained for backward compatibility only.

       For more robust and customizable output use findmnt(8), especially in your scripts.  Note that control characters in the mountpoint name are replaced with '?'.

       The following command lists all mounted filesystems (of type type):

              mount [-l] [-t type]

       The option -l adds labels to this listing.  See below.
```

Comment (MegaV0lt) [reply log2ram]: OK, changed: https://github.com/azlux/log2ram/pull/130/commits/6e110ae11c84353ee608e68269328d98b699a700

**Comment by azlux** [comment]
@zenofile has you said it's part of `util-linux`. Should we add it to the package depends ? Is there are a case/distrib with systemd and without the package ? (I personnaly don't find any)

**Comment by MegaV0lt** [comment]
If you löook here: https://de.wikipedia.org/wiki/Util-linux#Enthaltene_Programme
`mount` is aslo part of `util-linux`. so without `util-linux` the script should not work anyway...

**Comment by zenofile** [comment]
As @MegaV0lt correctly stated, there is no need to add this to the dependencies. There is no distribution not shipping `util-linux` by default.

**Pull request merged**